README.md

```diff
@@ -3224,6 +3229,425 @@ port-channel hash input offset. Valid values are integer or 'default'.
 port-channel symmetry hash. Valid values are true, false or 'default'.
 
 --
+### Type: cisco_route_map
+
+Manages a Cisco Route Map.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(1)        | 1.6.0                  |
+| N3k      | 7.0(3)I2(1)        | 1.6.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.6.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.6.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.6.0                  |
+| N9k-F    | 7.0(3)F1(1)        | 1.6.0                  |
+
+#### <a name="cisco_interface-caveats">Caveats</a>

```

Comment:
Change this to be `#### <a name="cisco_route_map-caveats">Caveats</a>` and make sure the link works.